packages/backend/models/unified/company.ts

```diff
@@ -1,10 +1,13 @@
+import { TP_ID } from '@prisma/client';
+import { PipedriveCompany } from '../../constants/pipedrive';
+
 export interface UnifiedCompany {
     name: string;
-    industry: string;
-    description: string;
-    annualRevenue: number;
+    industry: string; // not supported by pipedrive
```

Comment:
@Hvinder Lets add this comment about non-existing fields in the Fern `docs` property for these objects as well.